### PR TITLE
Remove Python 2.6 support.

### PR DESCRIPTION
**Pull request by jsha**
Fixes https://github.com/letsencrypt/letsencrypt/issues/515

cc @kuba because I know you've done a lot of work on maintaining the py26 support up to this point. I you haven't seen it yet, check out my comment on https://github.com/letsencrypt/letsencrypt/issues/515.

### CODE REVIEW

**Comment by schoen** [comment]
Looks good to me, though I don't know whether there will prove to be any other things that also need to be removed (or dependencies that could now be removed as a result of this change).

**Comment by kuba** [comment]
There haven't been any problems with Python 2.6 recently. I fixed most
recent problem (with mock, few weeks ago) in few minutes upon first failed
build. Why hurry so much if all is good?

**Comment by jsha** [comment]
Talking with James, Brad, and Seth, it sounds like Python 2.6 support has been a small but persistent backwards pull on their development time. I think it's safe to say team members mostly develop on Python 2.7, and then discover Python 2.6 regressions when running final cross-platform tests for submission. That sometimes involves going back and changing code to be Python 2.7 compatible.

With launch coming up in eight weeks, we really need to remove all stumbling blocks in the way of making fast forward progress. I think the work Python 2.6 generates vs the benefit I expect it to get us argues strongly in favor of cutting it. Like I said in #515, we'll be able to get much better data after launch, as we find out how many people want to use Let's Encrypt but are stymied by lack of Python 2.6 support.

**Comment by jsha** [comment]
Updated to remove py3+ as well, since the py3 tests are breaking in #653.

**Comment by bradmw** [comment]
LGTM!

**Comment by pde** [comment]
Merged for now; we can reenable python 2.6 support once we're in good launched shape!

**Comment by pde** [comment]
(Also disabling tox for these Python versions, which will sometimes mean faster turnarounds from Travis)

**Comment by kuba** [comment]
Urm...... https://github.com/letsencrypt/letsencrypt/pull/704

File: tox.ini [hunk 1]
```diff
@@ -40,7 +40,6 @@ commands =
     ./tox.cover.sh
 
 [testenv:lint]
-# recent versions of pylint do not support Python 2.6 (#97, #187)
```
Comment (kuba) [inline tox.ini]: could we please not remove this comment? I'm gonna have really hard times tracking this back when py 2.6 support will be back...

**Pull request merged**